docs/code_push/build.md

```diff
@@ -0,0 +1,26 @@
+---
+sidebar_position: 5
+title: 🔨 Build
+description: Learn how to generate release builds of your Flutter app.
+---
+
+# Build your Shorebird App
+
+You can use `shorebird build` to build a release version of your app including code push:
```

Comment:
`release` is generally the main flow, no?

Reply:
Yeah build is last in this section.